ethpm_types/contract_type.py

```diff
@@ -1,5 +1,6 @@
+from collections.abc import Callable, Iterable
 from functools import cached_property, singledispatchmethod
-from typing import Callable, Dict, Iterable, List, Optional, Tuple, Type, TypeVar, Union, cast
```

Comment:
I think `Type` is still in use

Reply:
You use lowercase `type` now , no import needed!